docs/viewlifecycle.md

```diff
@@ -66,8 +66,9 @@ part of the destruction lifecycle.
 |   1   |  `before:destroy` |       `view` - view being destroyed       |
 |       |                   | `...args` - arguments passed to `destroy` |
 |   2*  |  `before:detach`  |       `view` - view being detached        |
-|   3*  |  `detach`         |       `view` - view being detached        |
-|   4   |  `destroy`        |       `view` - view being destroyed       |
+|   3*  |  `dom:remove`     |       `view` - view being detached        |
```

Comment:
I think I get it all now. We'll need a separate ticket to improve the documentation in this area to simplify it.